airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/json_schema_helper.py

```diff
@@ -164,6 +164,22 @@ def get_expected_schema_structure(schema: dict, annotate_one_of: bool = False) -
     :returns list of object property keys paths
     """
     paths = []
+    if "$ref" in schema:
```

Comment:
hm, I'm not sure we should update SAT in this PR

Reply:
otherwise it wouldnt work with github schemas